Makefile

```diff
@@ -0,0 +1,449 @@
+# VERSION defines the project version for the bundle.
```

Comment:
only need to move the helm related make targets and the variables that those make targets use 

when we clone down awx-operator we can rename awx-operator's makefile as Makefile.awx-operator
than at the top of this Makefile do a include to combine the 2 Makefile